walkthroughs/week-4-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -15,9 +15,90 @@
 package com.google.sps;
 
 import java.util.Collection;
+import java.util.Comparator;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
 
 public final class FindMeetingQuery {
   public Collection<TimeRange> query(Collection<Event> events, MeetingRequest request) {
-    throw new UnsupportedOperationException("TODO: Implement this method.");
+    Collection<String> attendees = request.getAttendees();
+    long duration = request.getDuration();
+    boolean inattendance = false; 
+    
+    List<Event> newEvents = new ArrayList<Event> ();
+
+    if (duration > 1440)
```

Comment:
1440 is the number of minutes in the day, so that if statement is checking that the meeting request does not include a time duration that is over a day long.

Reply:
Can we have a final const for this?